galley/pkg/source/kube/schema/resourcespec.go

```diff
@@ -51,6 +48,36 @@ type ResourceSpec struct {
 
 	// The converter to use
 	Converter converter.Fn
+
+	// Versions of the K8s resource
+	Versions []CrdVersion
+
+	// Version of the K8s resource
+	Version string
+}
+
+type CrdVersion struct {
```

Comment:
Nit: i don't think you need CRD prefix. It is apparent from the package name/ambient context.